src/GafferScene/SceneAlgo.cpp

```diff
@@ -330,7 +330,7 @@ class CapturingMonitor : public Monitor
 			capturedProcess->type = process->type();
 			capturedProcess->plug = process->plug();
 			capturedProcess->destinationPlug = process->destinationPlug();
-			capturedProcess->context = new Context( *process->context() );
+			capturedProcess->context = new Context( *process->context(), /* canceller = */ nullptr );
```

Comment:
Do we need the same tweak here too: https://github.com/GafferHQ/gaffer/blob/6866f1f4e4221b4f6d540e798d70149f76fcb21e/src/GafferScene/SceneAlgo.cpp#L626

Reply:
Added one here : https://github.com/GafferHQ/gaffer/pull/3801/commits/5f14c092777f71170dbb10495ad7312a47cd70e0#diff-0fe233058240c09c22d20109c5370bfbR659-R662